src/main/java/org/jenkinsci/plugins/authorizeproject/strategy/SystemAuthorizationStrategy.java

```diff
@@ -0,0 +1,227 @@
+ * {@link org.jenkinsci.plugins.authorizeproject.GlobalQueueItemAuthenticator}
+ * is forcing jobs to a user other than {@link ACL#SYSTEM}.
+ *
+ * @since 1.1.1
+ */
+public class SystemAuthorizationStrategy extends AuthorizeProjectStrategy {
+
+    @DataBoundConstructor
+    public SystemAuthorizationStrategy() {
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public Authentication authenticate(Job<?, ?> job, Queue.Item item) {
+        return ACL.SYSTEM;
+    }
+
+    /**
+     * Return {@link SystemAuthorizationStrategy} configured in a job.
+     *
+     * @param job the {@link Job}
+     * @return the {@link SystemAuthorizationStrategy} or {@code null}
+     */
+    protected static SystemAuthorizationStrategy getCurrentStrategy(Job<?, ?> job) {
+        if (job == null) {
+            return null;
+        }
+
+        AuthorizeProjectProperty prop = job.getProperty(AuthorizeProjectProperty.class);
+        if (prop == null) {
+            return null;
+        }
+
+        AuthorizeProjectStrategy strategy = prop.getStrategy();
+        if (!(strategy instanceof SystemAuthorizationStrategy)) {
+            return null;
+        }
+
+        return (SystemAuthorizationStrategy) strategy;
+    }
+
+    /**
+     * Called when XSTREAM2 instantiates this from XML configuration.
+     *
+     * When configured via REST/CLI, {@link Descriptor#newInstance(StaplerRequest, JSONObject)} is not called.
+     * Instead checks authentication here.
+     *
+     * @return return myself.
+     * @throws IOException authentication failed.
+     */
+    private Object readResolve() throws IOException {
+        Jenkins instance = Jenkins.getInstance();
+        if (instance == null || !instance.hasPermission(Jenkins.ADMINISTER)) {
+            // This is called via REST/CLI.
+            // As REST/CLI interface saves configuration after successfully load object from the XML,
+            // this prevents the new configuration saved.
+            throw new IOException(Messages.SystemAuthorizationStrategy_readResolve());
+        }
+        return this;
+    }
+
+    /**
+     * For now we are an object with no configurable fields, so return a fixed value.
+     * If we add configurable fields we probably should consider removing the final.
+     *
+     * @return our hashCode.
+     */
+    @Override
+    public final int hashCode() {
+        return SystemAuthorizationStrategy.class.hashCode();
+    }
+
+    /**
+     * For now we are an object with no configurable fields, so strict instanceof establishes equality.
+     * If we add configurable fields we probably should consider removing the final.
+     *
+     * @param obj the object to test equality with.
+     * @return {@code true} if and only if this is a equivalent {@link SystemAuthorizationStrategy} instance.
+     */
+    @Override
+    public final boolean equals(Object obj) {
+        return obj != null && SystemAuthorizationStrategy.class == obj.getClass();
```

Comment:
The object has no fields, so any instance is equal to another... but only if they are the same class